examples/flush-notes/src/const.ts

```diff
@@ -0,0 +1,8 @@
+export const CONTENT_TOPIC = "/flush-notes/1/note/proto";
+
+export enum WakuStatus {
+  Initializing = "Initializing...",
+  WaitingForPeers = "Waiting for peers...",
+  Connected = "Connected",
+  Failed = "Failed to initialize(see logs)",
+}
```

Comment:
I agree about a need to cluster types in one place if they are generic, if they are strongly bound to one place - they should be located in that place (one file)

for `enum` it is tricky as it can be a type and can be a variable as it gets into end bundle and is not ephemeral as just `types`. so in this context it is a `const` as it was used in UI directly for rendering message